javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/bootstrap/internal/CommonConfig.java

```diff
@@ -5,18 +5,26 @@
 
 package io.opentelemetry.javaagent.bootstrap.internal;
 
+import static java.util.Arrays.asList;
 import static java.util.Collections.emptyList;
 import static java.util.Collections.emptyMap;
+import static java.util.Collections.unmodifiableList;
 
+import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 
 /**
  * This class is internal and is hence not for public use. Its APIs are unstable and can change at
  * any time.
  */
 public final class CommonConfig {
 
+  private static final List<String> DEFAULT_KNOWN_METHODS =
+      unmodifiableList(
+          asList("CONNECT", "DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT", "TRACE"));
```

Comment:
<grump> I don't like that this set is duplicated here and in `HttpRequestMethodUtil`. It makes it too easy down the road to change one and not the other. Is there common place that these can be located?